ext/dba/config.m4

```diff
@@ -640,34 +599,22 @@ if test "$PHP_CDB" = "yes"; then
   AC_DEFINE(DBA_CDB, 1, [ ])
   cdb_sources="libcdb/cdb.c libcdb/cdb_make.c libcdb/uint32.c"
   THIS_RESULT="builtin"
+  PHP_DBA_STD_RESULT
 elif test "$PHP_CDB" != "no"; then
-  PHP_DBA_STD_BEGIN
-  for i in $PHP_CDB /usr/local /usr; do
-    if test -f "$i/include/cdb.h"; then
-      THIS_PREFIX=$i
-      THIS_INCLUDE=$i/include/cdb.h
-      break
-    fi
-  done
-
-  if test -n "$THIS_INCLUDE"; then
-    for LIB in cdb c; do
-      PHP_CHECK_LIBRARY($LIB, cdb_read, [
-        AC_DEFINE_UNQUOTED(CDB_INCLUDE_FILE, "$THIS_INCLUDE", [ ])
-        AC_DEFINE(DBA_CDB, 1, [ ])
-        THIS_LIBS=$LIB
-      ], [], [-L$THIS_PREFIX/$PHP_LIBDIR])
-      if test -n "$THIS_LIBS"; then
-        break
-      fi
-    done
-  fi
-
-  PHP_DBA_STD_ASSIGN
-  PHP_DBA_STD_CHECK
-  PHP_DBA_STD_ATTACH
+  PKG_CHECK_MODULES([CDB], [cdb])
```

Comment:
Hm... this looks problematic.`--with-cdb` would give you the bundled library and `--with-cdb=foobar` will give you an external one (but with `foobar` ignored).

It would be best to unbundle `cdb`, but I have no idea what state it is in (wrt patches).

Reply:
For libgd and libpcre we use the `--with-external-gd` and `--with-external-pcre` options, so I guess it would make sense to use `--with-external-cdb` here.

Reply:
ok, I'll look at adding that in the next couple of days. Thanks!